docs/index.css

```diff
@@ -46,3 +46,51 @@ code {
 .Demo ul {
   margin: var(--space-large);
 }
+
+.Footer {
+  position: fixed;
+  bottom: 0;
+  width: 100%;
+  box-sizing: border-box;
```

Comment:
This can be removed since `box-sizing` is inherited 
<img width="1409" alt="Screen Shot 2022-09-15 at 12 38 23 PM" src="https://user-images.githubusercontent.com/3081483/190483658-4ed90756-c6da-4be3-9679-5f7f6450ab6b.png">